docs/npm_import.md

```diff
@@ -1,11 +1,11 @@
 <!-- Generated with Stardoc: http://skydoc.bazel.build -->
 
-Repository rules to fetch third-party npm packages
+Repository rules to declare npm packages
```

Comment:
technically the `npm_import` does the fetching and `npm_translate_lock` just declares a whole bunch of npm imports